example/MindSpore-example/mindspore_cpu/mindspore-cpu.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: mindspore-cpu
+spec:
+  minAvailable: 1
+  schedulerName: volcano
+  policies:
+    - event: PodEvicted
+      action: RestartJob
+  plugins:
+    ssh: []
+    env: []
+    svc: []
+  maxRetry: 5
+  queue: default
+  # Comment out the following section to enable volumes for job input/output.
+  #volumes:
+  #  - mountPath: "/myinput"
+  #  - mountPath: "/myoutput"
+  #    volumeClaimName: "testvolumeclaimname"
+  #    volumeClaim:
+  #      accessModes: [ "ReadWriteOnce" ]
+  #      storageClassName: "my-storage-class"
+  #      resources:
+  #        requests:
+  #          storage: 1Gi
+  tasks:
+    - replicas: 8
+      name: "pod"
+      template:
+        spec:
+          containers:
+            - command: ["/bin/bash", "-c", "python /tmp/lenet.py"]
+              image: lyd911/mindspore-cpu-example:0.2.0
+              imagePullPolicy: IfNotPresent
+              name: mindspore-cpu-job
+              resources:
+                limits:
+                  cpu: "1"
+                requests:
+                  cpu: "1"
+              volumeMounts:
+              - name: training-result
+                mountPath: /tmp/result
```

Comment:
nit: i donot think this path is used to store any output

Reply:
removed